qiskit/compiler/transpiler.py

```diff
@@ -453,7 +409,7 @@ def callback_func(**kwargs):
         optimization_method=optimization_method,
         _skip_target=_skip_target,
     )
-    out_circuits = pm.run(circuits, callback=callback)
+    out_circuits = pm.run(circuits, callback=callback, num_processes=num_processes)
```

Comment:
Do you think we should have a fix release note for this? This argument totally wouldn't ever work before, we were setting the flag in the single process path and we weren't setting the argument when the call could be multiprocess.

Reply:
Added in [250ff77](https://github.com/Qiskit/qiskit/pull/12183/commits/250ff7744f76c3d928748672d549a0865123e74c)